lib/sycamore/sycamore/tests/unit/query/test_schema.py

```diff
@@ -96,16 +136,6 @@ def test_opensearch_schema():
         }
     }
 
-    # Verify we can handle schemas where int and float are both used
-    airspeeds = [500, 37.5, 300, 217.11]
```

Comment:
I deliberately did int -> float -> int -> float to make sure it worked in all cases. 
I prefer the structure where I can see all the related values for a single field in a single variable rather than spread across all of the examples because it makes it easier to tell if it's doing the right thing. 
I'm ok either way on this choice, so this comment is an FYI.

Reply:
Understood, I wanted there to be consistency in how the examples were being constructed.

Reply:
I can add back an int / float / int / float case.